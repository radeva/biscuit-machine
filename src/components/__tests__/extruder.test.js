import React from 'react';
import Extruder from '../extruder';
import { render, cleanup } from '@testing-library/react';

afterEach(cleanup);

describe('Extruder', () => {
  it('should render correctly', () => {
    const { container } = render(<Extruder />);

    expect(container).toMatchSnapshot();
  });
});
