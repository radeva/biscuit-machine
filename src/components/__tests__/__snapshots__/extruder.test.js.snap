// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Extruder should render correctly 1`] = `
<div>
  <div
    class="extruder left"
  >
    <svg>
      extruder.svg
    </svg>
  </div>
</div>
`;
